AEROFLEX/iocBoot/iocAEROFLEX-IOC-01/config.xml

```diff
@@ -6,6 +6,7 @@
 <macros>
 <macro name="PORT" pattern="^COM[0-9]+$" description="Serial COM port" hasDefault="NO" />
 <macro name="DEV_TYPE" pattern="^(2023A|2030)$" description="Type of Aeroflex" defaultValue="2023A" hasDefault="YES" />
+<macro name="RF_PREC" pattern="^(6|2)$" description="RF level decimal precision" defaultValue="6" hasDefault="YES" />
```

Comment:
I can see this being an issue. Let's say I change my device type but forget to adjust the precision. Is something going to break? Ideally the precision should depend on the device type.
Using a `scalcout` record on the `DEV_TYPE` macro should work.

Reply:
I'd like to set the value of `RF_PREC` based on the value of `DEV_TYPE`. Can this be done with `st-common.cmd`?

Reply:
You should be able to use an `if` statement in `st-common.cmd` on the device type macro, and then depending on that result set a "precision" variable and pass that as the precision macro value.

Reply:
I tried adding the following line to `st-common.cmd`:
`if $(DEV_TYPE)==2023A (SET rf_prec = 6) else (SET rf_prec = 2)`
And I get the following error:
`st-common.cmd line 31: Command if not found.`